learn/multi_search/multi_search_vs_federated_search.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Differences between multi-search and federated search — Meilisearch API reference
+description: This article defines multi-search and federated search and then describes the different uses of each.
+---
+
+# Differences between multi-search and federated search
+
+This article defines multi-search and federated search and then describes the different uses of each.
+
+## What is multi-search?
+
+Multi-search, also called multi-index search, is a search operation that queries two or more indexes at the same time. Meilisearch then returns a separate list results for each index. Use the `/multi-search` route to perform multi-searches.
+
+Multi-search favors discovery scenarios, where users might not have a clear idea of what they need and searches might have many valid results.
+
+## What is federated search?
+
+Federated search is a type of multi-index search. This operation also queries multiple indexes, but returns a single list with the most relevant results from all queried indexes. Use the `/multi-search` route and specify a non-null value for `federation` to perform a federated search.
```

Comment:
Similarly, this describes the use case of merging the query `toto` across multiple indexes, but federated search can be about merging the results of the `toto` and the `titi` queries against the same index as well.

Reply:
I am currently using version 0.43 on nextjs 14. but it seems `multi-search` does not have `federation` in it. is there anyway i can do this on v0.43

Reply:
**endPoint:** `/multi-search`
**method:** `POST`

**Request:** 
```
{
	"federation": {
		"offset": 0,
		"limit": 10
	},
	"queries": [
		{
			"indexUid": "services",
			"q": "app"
		},
		{
			"indexUid": "tasks",
			"q": "app"
		}
	]
}
```

**Response:**
```
{
	"message": "Unknown field `federation`: expected one of `queries`",
	"code": "bad_request",
	"type": "invalid_request",
	"link": "https://docs.meilisearch.com/errors#bad_request"
}
```

Reply:
Hello @asinan007, this PR is not the place for these questions related to SDKs. I suggest you open an issue on the relevant SDK repository.